fuse/main.c

```diff
@@ -21,7 +21,9 @@
 */
 
 #include <exfat.h>
+#ifndef FUSE_USE_VERSION
```

Comment:
Why is this ifndef needed? It's a bit scaring to give anyone the ability to override FUSE API version.